keps/sig-scheduling/5075-dra-consumable-capacity/kep.yaml

```diff
@@ -0,0 +1,47 @@
+title: DRA: Consumable Capacity
+kep-number: 5075
+authors:
+  - "@sunya-ch"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-node
+  - sig-network
+status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced
+creation-date: 2025-30-01
+reviewers:
+  - "@johnbelamaric"
+  - "@catblade"
+approvers:
+  - TBD
+
+see-also:
+#  - "/keps/sig-aaa/1234-we-heard-you-like-keps"
+#  - "/keps/sig-bbb/2345-everyone-gets-a-kep"
+# replaces:
+#   - "/keps/sig-ccc/3456-replaced-kep"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha|beta|stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.34"
+  beta: "v1.35"
+  stable: "v1.36"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: DynamicDeviceProvision
+    components:
+      - kube-scheduler
```

Comment:
```suggestion
      - kube-scheduler
      - kubelet
```

Reply:
Add also `kube-apiserver` for the validation and for the fields which must be dropped if the feature-gate is disabled.